shopping-cart/deploy/scripts/common/setupEnv.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
+## The ID will be the PR number, the branch name on Travis or `local`
+if [[ -n "${TRAVIS_PULL_REQUEST}" ]]
+then
+    ID="${TRAVIS_PULL_REQUEST}" # Pull request number
```

Comment:
This case probably isn't needed anymore, since I also changed `.travis.yml` to only deploy on branch pushes, but I thought I'd keep it just in case PR builds are ever enabled again in the future. `TRAVIS_BRANCH` is the target branch for pull request builds, not the origin branch, so it could cause a conflict to use that.